.travis.yml

```diff
@@ -19,12 +19,11 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DIST=trusty
-        - DIST=xenial
+        - DMD='2.070.2-*' DIST=xenial
```

Comment:
This wildcarding of version is not supported yet.

Reply:
It is done by `apt` itself. I can't find reference in documentation but I did some manual experiments and am reasonably sure it just instals latest available package where version string matches.

Reply:
OK, this is very interesting and useful. I was just playing with pinning and repo priorities and it seems to work well with that too. I guess we can move forward with it!